docs/sdk_performance.md

```diff
@@ -0,0 +1,57 @@
+# SDK Performance
+
+## SDK Metrics
+
+| Size   | RAM overload at peak | Batch average size | Batch max frequency |
+|--------|----------------------|--------------------|---------------------|
+| 612 KB | 6%                   | 10000 KB           | 1 second            |
+
+### Notes
+
+1. The SDK size was measured by computing the difference between the sizes of our sample application with and without
+the Datadog SDK dependency. Please have in mind that this number is volatile and it takes into 
+account also the SDK transitive dependencies. It can go up or down if your app has or not transitive
+dependencies in common with the SDK. For now the SDK uses the following transitive dependencies: 
+    -   androidx.core:core
+    -   androidx.navigation:navigation-fragment-ktx
+    -   androidx.navigation:navigation-ui-ktx
+    -   androidx.navigation:navigation-runtime-ktx
+    -   androidx.recyclerview:recyclerview
+    -   androidx.work:work-runtime
+    -   com.google.code.gson:gson
+    -   com.lyft.kronos:kronos-android
+    -   com.squareup.okhttp3:okhttp
+    -   org.jetbrains.kotlin:kotlin-stdlib
+     
+2. For the RAM overload value we profiled a sample application with/without the Datadog SDK features fully
```

Comment:
the RAM overload isn't that relevant. We don't know how many traces, RUM events and logs that covers. Also I'm not sure it should be a %. If an app A uses 128M and an app B uses 32M, if you add the SDK to both and send the same number of events, in theory app A won't use more RAM than app B. Or you're talking about the % of the device's RAM which doesn't make sense either.

Reply:
I think this is too volatile same as for the batch size. I don't think we will add this after all.